ctapipe/image/ellipsoid.py

```diff
@@ -0,0 +1,443 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: UTF-8 -*-
+"""
+Shower image parametrization based on image fitting.
+"""
+
+from enum import Enum
+
+import astropy.units as u
+import numpy as np
+from astropy.coordinates import Angle
+from iminuit import Minuit
+
+from ctapipe.image.cleaning import dilate
+from ctapipe.image.hillas import hillas_parameters
+from ctapipe.image.pixel_likelihood import (
+    mean_poisson_likelihood_gaussian,
+    neg_log_likelihood_approx,
+)
+from ctapipe.image.toymodel import SkewedGaussian, SkewedGaussianLaplace
+
+from ..containers import ImageFitParametersContainer
+
+PED_TABLE = {
+    "LSTCam": 2.8,
+    "NectarCam": 2.3,
+    "FlashCam": 2.3,
+    "SST-Camera": 0.5,
+    "CHEC": 0.5,
+    "DUMMY": 0,
+    "testcam": 0,
+}
+SPE_WIDTH = 0.5
+
+__all__ = [
+    "image_fit_parameters",
+    "ImageFitParameterizationError",
+    "PDFType",
+]
+
+
+class PDFType(Enum):
+    gaussian = "gaussian"
+    laplace = "laplace"
+    skewed = "skewed"
+
+
+def create_initial_guess(geometry, image, size):
+    """
+    This function computes the seeds of the fit with the Hillas parameters
```

Comment:
I think you need empty lines before headings in the docstrings. At least it is much more readable